restlicsharpclient/restliclient/util/RequestUrlBuilder.cs

```diff
@@ -39,35 +40,24 @@ public RequestUrlBuilder(Request<TResponse> request, string urlPrefix)
 
         public string Build()
```

Comment:
As we discussed, we may want to return `uri` here instead of string to get early validation to fail fast instead of delaying it later to error out in sending request.